src/languageSupport/languages/flux/lsp/utils.ts

```diff
@@ -22,6 +24,15 @@ const createNotification = (method: string, params: object = {}) => {
   }
 }
 
+const createRequest = (method: string, params: object = {}) => {
+  return {
+    jsonrpc: JSONRPC,
+    method,
+    params,
+    id: Math.random().toString(),
```

Comment:
Requirement is for a unique ID.
Does not need to be in the same monotonically increasing order as the unique ID from the monaco-editor-initiated events.